packages/core/src/services/providerLogs/getInitialForPreview.ts

```diff
@@ -0,0 +1,16 @@
+import { ProviderLogDto } from '../../browser'
+import { findLastProviderLogFromDocumentLogUuid } from '../../data-access'
+import { buildProviderLogResponse } from './buildResponse'
+
+export async function getInitialProviderLogForPreview({
```

Comment:
A bit weird this name. Is used for other places other than preview no?

Reply:
will change, vibe coding